extensions/amp-story-auto-analytics/amp-story-auto-analytics.md

```diff
@@ -37,3 +37,37 @@ The `amp-story-auto-analytics` component configures [`amp-analytics`](https://am
 ## gtag-id (required)
 
 Google Analytics ID used to link the story to the analytics account.
+
+# Triggers
+
+The following analytics triggers are dispatched when using the `amp-story-auto-analytics` component:
+
+## `storyPageCount`
+
+Fired once per story view. It includes the following parameters:
+
+| Parameter             | Value                                                           |
+| --------------------- | --------------------------------------------------------------- |
+| `ea` (event_action)   | `story_page_count`                                              |
+| `ec` (event_category) | Title of the story (`<title>` tag in the HTML of the Web Story) |
+| `el` (event_label)    | Length of the story in number of pages                          |
+
+## `storyEnd`
+
+Fired once the last page of the story has been viewed. It includes the following parameters:
+
+| Parameter             | Value                                                           |
+| --------------------- | --------------------------------------------------------------- |
+| `ea` (event_action)   | `story_complete`                                                |
+| `ec` (event_category) | Title of the story (`<title>` tag in the HTML of the Web Story) |
+| `el` (event_label)    | Title of the story (`<title>` tag in the HTML of the Web Story) |
+
+## `storyPageIndex`
+
+Fired once per page once it has been viewed. It includes the following parameters:
```

Comment:
Nit: "once" is repeated twice, we can say "fired once a page has been viewed for the first time" or similar